src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -517,17 +521,11 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
           <TabList
             onChange={handleTabChange}
             aria-label='inspector tabs'
+
           >
-            <Tab
-              label={getTranslation('properties', props.language)} {...a11yProps(0)}
-            />
-            <Tab
-              label={getTranslation('restrictions', props.language)} {...a11yProps(1)}
-            />
-            <Tab
-              hidden={itemToDisplay?.type !== 'object'}
-              label={getTranslation('fields', props.language)} {...a11yProps(2)}
-            />
+            {renderTab('properties', 0)}
+            {renderTab('restrictions', 1)}
+            {renderTab('fields', 2, itemToDisplay?.type !== 'object')}
```

Comment:
I like the reduction in lines, but I think that we should try to keep the markup as close to pure markup as possible;
Why not make it so the markup is more like this:
```jsx
<Tab labelId='properties' tabIndex={0} />
```

Reply:
done